### DISCUSSION TITLE
Consider removing the documtation of unfrozen German or Polish locales

### DISCUSSION THREAD

**Discussion by yin1999** [mdn/mdn-community#391] [Localization] [answered]
In PEERS_GUIDELINES of translated-content, we just mentioned that "we could specify a locale listed [here](https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md#your-locale) to discuss about unfrozen. But we have [retired/archived those two locales last year](https://github.com/orgs/mdn/discussions/221), as they are "so out of date". And those contents have been removed from translated-content.

Should we remove the documation of unfrozen German or Polish locales from PEERS_GUIDELINES in translated-content?

**Comment by Rumyra** [comment]
Yes we aboslutely should @yin1999 - thank you for flagging this. Do you want to go ahead?

**Reply by yin1999** [reply]
Thanks for the answer @Rumyra. I'd like to take this, but I'm not sure about the policy of "Adding/Activating a new locale other than those two retired locales". Is it possible that MDN would accepting a new locale in translated-content? If so, we only need to rewrite the [Your locale](https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md#your-locale) section; if not, we may need to delete most of the relevant content in the guidelines.

**Reply by Rumyra** [reply, marked as answer]
Let's delete the relevant content - I believe we have enough channels open now (both here on github and discord) that folks can get in touch with interest without having this specified guidance 👍

**Comment by pransh15** [comment]
Closing this as answered.
